clients/da-vinci-client/src/main/java/com/linkedin/davinci/storage/StorageService.java

```diff
@@ -172,6 +172,17 @@ private void restoreAllStores(
         AbstractStorageEngine storageEngine;
 
         try {
+          String store = Version.parseStoreFromVersionTopic(storeName);
```

Comment:
Can we move these lines up to line 168 so we even don't have to load the config for it?

Reply:
I didnt do that as wanted a smaller try block.  But sure.